ante/testutils/testutil.go

```diff
@@ -98,7 +90,31 @@
 
 	suite.Require().NotNil(suite.network.App.AppCodec())
 
-	options := chainante.HandlerOptions{
+	chainConfig := evmtypes.DefaultChainConfig(suite.network.GetChainID())
+	if !suite.enableLondonHF {
+		maxInt := sdkmath.NewInt(math.MaxInt64)
+		chainConfig.LondonBlock = &maxInt
+		chainConfig.ArrowGlacierBlock = &maxInt
+		chainConfig.GrayGlacierBlock = &maxInt
+		chainConfig.MergeNetsplitBlock = &maxInt
+		chainConfig.ShanghaiBlock = &maxInt
+		chainConfig.CancunBlock = &maxInt
+	}
+
+	// get the denom and decimals set when initialized the chain
+	// to set them again
+	// when resetting the chain config
+	denom := evmtypes.GetEVMCoinDenom()       //nolint:staticcheck
+	decimals := evmtypes.GetEVMCoinDecimals() //nolint:staticcheck
+	configurator := evmtypes.NewEVMConfigurator()
+	configurator.ResetTestConfig()
+	err := configurator.
+		WithChainConfig(chainConfig).
+		WithEVMCoinInfo(denom, uint8(decimals)).
+		Configure()
```

Comment:
The codeQL is reading the ResetTestConfig without the `test` tag and that's why this is unreachable. But when compiling the code with the correct flag, this will be executed.